test/integration/global_web_socket_monitor/sockjs_port_test.dart

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@Tags(const ['no-dart2'])
+@Tags(['no-dart2'])
```

Comment:
Hmm. Can we run these tests in Dart 2 now? Also, it looks like maybe we should check to see if the `dart_test.yaml` file needs updating.